src/dotty/tools/dotc/transform/ElimRepeated.scala

```diff
@@ -43,6 +43,8 @@ class ElimRepeated extends MiniPhaseTransform with InfoTransformer with Annotati
           paramTypes.init :+ last
         } else paramTypes
       tp.derivedMethodType(paramNames, paramTypes1, resultType1)
+    case tp: ExprType if tp.resultType.isRepeatedParam =>
+       ExprType(tp.resultType.underlyingIfRepeated(isJava = false))
```

Comment:
https://github.com/scala/scala/blob/2.11.x/src/library/scala/StringContext.scala#L54

Accessor `parts` gets type `ExprType(Repeated[String])`. Where would you propose to handle such types? They should not survive until erasure, as erasure will erase them to `<repeated>` which is wrong.